plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/common/viewsupport/StatusInfo.java

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
This should not be required, given we already have Outcome and OutcomeFactory in the komodo spi